modules/concepts/hooks/list-of-hooks/actionAddressGridDefinitionModifier.md

```diff
@@ -0,0 +1,27 @@
+---
+Title: actionAddressGridDefinitionModifier
+hidden: true
+hookTitle: 'Modify address grid definition'
+files:
+    -
+        url: 'https://github.com/PrestaShop/PrestaShop/blob/9.0.x/'
```

Comment:
I think it could be interesting to have an origin of this hook, it fits the format of action<GridId>GridDefinitionModifier and we know where the hook is.

Reply:
and if it is not possible or hard to achieve, then maybe we don't need to list all of them but improve the search mechanism further according to this information:

```
Search tip: Some hooks are generated dynamically, so their names are documented in a generic way.

For example, actionAdminCustomersFormModifier is documented as action<AdminControllerClassName>FormModifier. A regex based search has been implemented, and generic hooks should be matched. However, when you see a controller name or action in the hook name and you can’t find it, try searching for a part of the hook name, like FormModifier.
```

from what I see most of the hooks that were generated are action hooks with the dynamic names